libs/pageserver_api/src/models.rs

```diff
@@ -162,6 +162,14 @@ impl std::fmt::Debug for TenantState {
     }
 }
 
+#[serde_as]
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct LsnLease {
+    #[serde(rename = "valid_until_millis_since_epoch")]
+    #[serde_as(as = "serde_with::TimestampMilliSeconds")]
+    pub valid_until: SystemTime,
```

Comment:
I wonder if an rfc3339 would be more in line (not that we are able to always produce an rfc3339)?

Reply:
Oh, also, the serialization is now non-deterministic.

Reply:
Ok, I was thinking the `serde_with::TimestampMilliSeconds` was doing `SystemTime::now()` but it does not.

Reply:
I'd be okay with rfc3339 as well, we use it in the lsn<-> timestamp code. I suppose I can also use `format_rfc3339_millis`, shrug.

Reply:
added RFC 3339 format. Using milliseconds precision instead of the provided default.